cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -466,13 +538,15 @@ body depth_search recurse_basedir_exclude(d)
 ############################################################################
 
 body file_select cfe_internal_docroot_perms
-# @brief Select files not named `.htacces` or `settings.ldap.php`
+# @brief Select files (not dirs) not named `.htacces` or `settings.ldap.php`
```

Comment:
Might as well fix the *.htacces**s*** name.

Reply:
ACK.